pkg/machine/applehv/claim.go

```diff
@@ -1,5 +1,6 @@
-//go:build darwin && arm64
-// +build darwin,arm64
+//go:build (arm64 || amd64) && darwin
```

Comment:
There are no other arches for macos right? So only using the darwin tag seems simpler to me.

Reply:
right, duh